config/routes.rb

```diff
@@ -52,7 +52,9 @@
   end
 
   # Current redirection so that everyone has time to update bookmarks
+  get '/ausleihe', to: redirect { |params, request| "/internal/ausleihe/" }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unused block argument - `params`. You can omit all the arguments if you don't care about them.](https://www.codacy.com/app/lippertsjan/klausurtool-ror/file/2383456470/issues/source?bid=3141297&fileBranchId=3226894#l55)